desktop/src/main/java/bisq/desktop/main/overlays/windows/DisputeSummaryWindow.java

```diff
@@ -673,6 +673,13 @@ private void addButtons(Contract contract) {
     }
 
     private void showPayoutTxConfirmation(Contract contract, DisputeResult disputeResult, ResultHandler resultHandler) {
+        if (dispute.isPayoutDone()) {
+            new Popup().headLine(Res.get("disputeSummaryWindow.close.alreadyPaid.headline"))
```

Comment:
The `payoutDone` guard works fine, and it issues the log warning about avoiding double payouts.  However I cannot get this warning popup to show - so it seems redundant.

Reply:
You get this warning if you try to do a payout on the same user twice without closing the counterparty ticket first.